modules/apps/commerce/commerce-product-asset-categories-web/src/main/java/com/liferay/commerce/product/asset/categories/web/internal/portlet/action/EditAssetCategoryFriendlyURLMVCActionCommand.java

```diff
@@ -119,7 +118,7 @@ protected void doProcessAction(
 
 			String urlTitle = urlTitleMap.get(locale);
 
-			if (Validator.isNotNull(urlTitle)) {
+			if (urlTitle != null) {
```

Comment:
Hey @brianikim this change is fine, but you need to add some logic to make sure:

1. the default language translation is not deleted.
2. you don't put an empty string in DB, but you actually delete that record.

Reply:
Hello @andrea-ale-sbarra , thank you for pointing that out. The updated PR should have all the changes you have requested.